routes/dms.js

```diff
@@ -34,12 +34,10 @@ module.exports = function () {
   // -----------------------------------------------
 
   router.get('/', async (req, res) => {
-    let posts = res.locals.posts || []
```

Comment:
@sagargg I totally agree that this is redundant and we should remove it but why are you doing it in this PR?